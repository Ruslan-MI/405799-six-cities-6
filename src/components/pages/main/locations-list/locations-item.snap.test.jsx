import React from "react";
import {
  render,
} from "@testing-library/react";
import LocationsItem from "./locations-item";
import {
  StoreNameSpace,
  CITIES,
} from "../../../../const";
import Test, {
  mockMainPageInitialState,
} from "../../../../mocks-for-test";

it(`Locations item should render correctly`, () => {
  const dataForStore = {
    [StoreNameSpace.MAIN_PAGE]: mockMainPageInitialState
  };

  const {
    container,
  } = render(<Test dataForStore={dataForStore}>
    <LocationsItem city={CITIES[0]} />
  </Test>);

  expect(container).toMatchSnapshot();
});
