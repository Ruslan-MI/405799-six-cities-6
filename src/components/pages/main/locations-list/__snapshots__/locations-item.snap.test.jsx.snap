// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Locations item should render correctly 1`] = `
<div>
  <li
    class="locations__item"
  >
    <a
      class="locations__item-link tabs__item tabs__item--active"
      href="#"
    >
      <span>
        Paris
      </span>
    </a>
  </li>
</div>
`;
